comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportClient.scala

```diff
@@ -31,9 +31,10 @@ class GrpcTransportClient(
     maxMessageSize: Int,
     packetChunkSize: Int,
     tempFolder: Path,
-    clientQueueSize: Int
+    clientQueueSize: Int,
+    parallelism: Int
 )(
-    implicit scheduler: Scheduler,
+    implicit ioScheduler: Scheduler,
```

Comment:
Maybe it makes sense to keep only main scheduler as implicit and others as explicit parameters. It's important thing to be configured correctly and also it's already passed explicitly.